### PR TITLE
TD-124: Setup CI/CD with GH Actions

### PR DESCRIPTION
**Pull request by keynslug**
* Pull recently implemented features
* Refactor for the elvis pleasure
* Drop client URL logging
* Drop geo handler and tests (TD-137)

### CODE REVIEW

**Review by kehitt — APPROVED** [review]
🏁

File: rebar.config [hunk 1]
```diff
@@ -27,32 +27,33 @@
...
-    {prometheus_cowboy, "0.1.8"},
-    {bouncer_proto, {git, "https://github.com/rbkmoney/bouncer-proto.git", {branch, master}}},
-    {bouncer_client, {git, "https://github.com/rbkmoney/bouncer_client_erlang.git", {branch, master}}},
-    {token_keeper_client, {git, "https://github.com/rbkmoney/token-keeper-client.git", {branch, master}}},
-    {party_client, {git, "https://github.com/rbkmoney/party_client_erlang.git", {branch, master}}},
-    {payout_manager_proto, {git, "https://github.com/rbkmoney/payout-manager-proto.git", {branch, master}}},
-    {how_are_you, {git, "https://github.com/rbkmoney/how_are_you.git", {branch, master}}},
-    {feat, {git, "https://github.com/rbkmoney/feat.git", {branch, master}}}
+    %% NOTE
+    %% Pinning to version "1.11.2" from hex here causes constant upgrading and recompilation of the entire project
```
[the remark this answers is not in the document]
Comment (keynslug) [reply rebar.config:31]: Скорее всего [вот с этим](https://github.com/potatosalad/erlang-jose/pull/122).

File: apps/capi/src/capi_handler_encoder.erl [hunk 2]
```diff
@@ -205,7 +205,7 @@ encode_invoice_line_meta(Line) ->
 
 encode_invoice_line_tax_mode(#{<<"type">> := <<"InvoiceLineTaxVAT">>} = TaxMode) ->
     %% for more info about taxMode look here:
-    %% https://github.com/rbkmoney/starrys/blob/master/docs/settings.md
+    %% https://github.com/valitydev/starrys/blob/master/docs/settings.md
```
[the remark this answers is not in the document]
Comment (keynslug) [reply apps/capi/src/capi_handler_encoder.erl]: 🌯

File: elvis.config [hunk 3]
```diff
@@ -1,33 +1,66 @@
...
-                    {elvis_text_style, no_tabs},
-                    {elvis_text_style, no_trailing_whitespace},
-                    {elvis_style, macro_module_names},
-                    {elvis_style, operator_spaces, #{rules => [{right, ","}, {right, "++"}, {left, "++"}]}},
-                    {elvis_style, nesting_level, #{level => 4}},
-                    {elvis_style, god_modules, #{
-                        limit => 30,
+                    %% Common settings
+                    {elvis_text_style, line_length, #{limit => 120}},
+                    {elvis_text_style, no_trailing_whitespace, #{ignore_empty_lines => true}},
```
[the remark this answers is not in the document]
Comment (keynslug) [reply elvis.config]: 🌚

**Pull request merged**